Reference quirk. For `\w ആകാശവും|lemma="..." ...\w*` Paratext writes a
space at the end of the word (`ആകാശവും </char>`) that is not in the source,
and only when the attribute list has named attributes: verses 3 and 4, with
a bare default value, have no such space. The parser keeps the text as
written.

--- origin.xml
+++ origin.xml
@@ -3,6 +3,6 @@
   <para style="mt1">ഉല്പത്തിപുസ്തകം</para>
   <chapter number="1" style="c" sid="GEN 1" />
   <para style="p">
-    <verse number="1" style="v" sid="GEN 1:1" />ആദിയിൽ ദൈവം <char style="w" lemma="ആകാശം" strong="l" x-morph="He,R:Sp1cs">ആകാശവും </char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:1" /><verse number="2" style="v" sid="GEN 1:2" />ആദിയിൽ ദൈവം <char style="w" lemma="" strong="l" x-morph="He,R:Sp1cs">ആകാശവും </char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:2" /><verse number="3" style="v" sid="GEN 1:3" />ആദിയിൽ ദൈവം <char style="w" lemma="&quot;ആകാശം&quot;">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:3" /><verse number="4" style="v" sid="GEN 1:4" />ആദിയിൽ ദൈവം <char style="w" lemma="ആകാശം">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു.<verse eid="GEN 1:4" /></para>
+    <verse number="1" style="v" sid="GEN 1:1" />ആദിയിൽ ദൈവം <char style="w" lemma="ആകാശം" strong="l" x-morph="He,R:Sp1cs">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:1" /><verse number="2" style="v" sid="GEN 1:2" />ആദിയിൽ ദൈവം <char style="w" lemma="" strong="l" x-morph="He,R:Sp1cs">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:2" /><verse number="3" style="v" sid="GEN 1:3" />ആദിയിൽ ദൈവം <char style="w" lemma="&quot;ആകാശം&quot;">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു. <verse eid="GEN 1:3" /><verse number="4" style="v" sid="GEN 1:4" />ആദിയിൽ ദൈവം <char style="w" lemma="ആകാശം">ആകാശവും</char> ഭൂമിയും സൃഷ്ടിച്ചു.<verse eid="GEN 1:4" /></para>
   <chapter eid="GEN 1" />
 </usx>
\ No newline at end of file
